ocaml/src/cli_bench_common.ml

```diff
@@ -45,6 +45,13 @@ let prefix default =
        & info ["prefix"] ~docv:"prefix" ~doc
   )
 
+let process_prefix default =
+  let doc = "$(docv) to keep multiple bench processes out of each other's way" in
```

Comment:
Indeed: every process will have one file per client, but if you have multiple processes, they will overwrite the files. Hence the prefix. I'll try to think of something.